crates/html-macro/src/parser.rs

```diff
@@ -61,13 +61,14 @@ impl HtmlParser {
                 // The root node is named `node_0`. All of it's descendants are node_1.. node_2.. etc.
                 // This just comes from the `idx` variable
                 // TODO: Not sure what the span is supposed to be so I just picked something..
-                let var_name = Ident::new(format!("node_{}", idx).as_str(), name.span());
+                let var_name_node = Ident::new(format!("node_{}", idx).as_str(), name.span());
+                let var_name_element_node = Ident::new(format!("element_node_{}", idx).as_str(), name.span());
                 let html_tag = format!("{}", name);
 
-                let node = quote! {
-                    let mut #var_name = VirtualNode::new(#html_tag);
+                let element_node = quote! {
```

Comment:
So now we make an `Element` and then later create a `VirtualNode` from it? Cool cool